numba/tests/test_parfors.py

```diff
@@ -434,6 +436,23 @@ def test_impl(v1, v2, m1, m2):
             return 4 * np.sum(x**2 + y**2 < 1) / 10
         self.check(test_impl, *self.simple_args)
 
+    @skip_unsupported
+    def test_randoms(self):
+        def test_impl(n):
+            A = np.random.standard_normal(size=(n, n))
+            B = np.random.randn(n, n)
+            C = np.random.normal(0.0, 1.0, (n, n))
+            D = np.random.chisquare(1.0, (n, n))
+            E = np.random.randint(1, high=3, size=(n, n))
+            F = np.random.triangular(1, 2, 3, (n, n))
+            return np.sum(A+B+C+D+E+F)
+
+        n = 128
+        # only run parallel version and don't compare results since random
+        # numbers generated in parallel are different
```

Comment:
Whilst not ideal, could an upper bound be calculated to assert some degree of validity on the answer?